workspace/lib/lita/handlers/teamcity.rb

```diff
@@ -494,6 +511,31 @@ def fetch_builds(build_url)
         return data
       end
 
+      def fetch_successful_builds(build_id_wildcard, response)
+        response_str = ''
+        count_success = 0
+        maximum_builds = 1000
+        end_time_teamcity, start_time_teamcity = get_teamcity_date_range()
+        current_day_teamcity = end_time_teamcity.split('T')[0]
+        url = "#{config.site}/app/rest/builds/?locator=status:SUCCESS,"\
+              "branch:(default:any),"\
+              "sinceDate:#{start_time_teamcity},"\
+              "untilDate:#{end_time_teamcity},count:#{maximum_builds}"
```

Comment:
why use untilDate? The sinceDate in the Curl should just be today's date at midnight.  No need to use untilDate.